.github/workflows/build.yml

```diff
@@ -31,9 +34,8 @@ jobs:
         if: failure()
         uses: rtCamp/action-slack-notify@master
         env:
-           SLACK_CHANNEL: getintoteaching_tech
```

Comment:
How does this find which channel to publish to? is the `getintoteaching_tech` the default one?